velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -54,7 +54,12 @@ int main(int argc, char** argv) {
       "chr",
       "replace",
       "might_contain",
-      "unix_timestamp"};
+      "unix_timestamp",
+      // Skip concat_ws due to the below issue:
```

Comment:
Without fuzzer coverage it will be hard to ensure the implementation is correct.